commands/backfill.js

```diff
@@ -66,13 +66,22 @@ module.exports = function container (get, set, clear) {
             var opts = {product_id: product_id}
             if (mode === 'backward') {
               opts.to = marker.from
+              // bitfinex v2 api required timestamp
+              if (exchange_id === 'bitfinex' && !marker.from){
```

Comment:
Thank you
if opts.to is null , bitfinex response 1 years ago data.
hm.. I'm thinking..